docs/api/mount.md

```diff
@@ -116,7 +116,7 @@ describe('Foo', () => {
   it('renders a div', () => {
     const wrapper = mount(Foo, {
       stubs: {
-        Bar: '<div class="stubbed" />',
+        Bar: '<div class="stubbed" />', // DEPRECATED
```

Comment:
if it is deprecated should we just nuke it from the docs?

Reply:
I'd make an argument against that as the typescript definitions show strings as a valid option.  It's very frustrating when you see something in the code, but missing in the documentation.  Is there a better place to indicate that strings are deprecated?

Reply:
Right, makes sense. Not sure.... strings aren't really deprecated, just ones that are valid HTML. Could we just add a check and throw and error if the string includes `<` or `>`?

Reply:
If you were to just pass text then the vue template compiler will complain about it not being a valid template.

Reply:
Right, I wasn't thinking. You'd need to pass `h('text')`.

Let's just drop it, and remove `string` from the TS definitions if it's not longer valid? Or am I missing something 🤔

Reply:
I've seen projects use it in the past.  To provide a migration path I'd say we keep it in until 1.0 is official.  That way a major version change will help manage a good point to make a breaking change.  Thoughts on that?

Reply:
I think making breaking changes **before** 1.0 make a lot more sense - once we hit 1.0, deprecating things becomes a lot more difficult.

What do you think if we just deprecate it right now and add a check + warning "This is now deprecated"?

Either way let's make a decision and get this merged up! I'll leave it up to you to make the final call.

Reply:
Deprecated as in still works for maybe the next release or two.  then rip it out?  I'm good with that.  Really it's just to give people a chance to migrate without completely breaking any of these stubs.

Completely agree with after 1.0 it's a pain to get something out.  I'd suggest that we remove this completely before 1.0 final release.  Any idea on how many more beta's there will be?

Reply:
Just show the right methods here, and add a note below, of what used to work, but is now deprecated. 

New users get to read the valid syntax, and those that are searching will find why its erroring out when they update :)

Reply:
Sounds good. Can we get this added and merge this @Stoom , thanks for sticking with this PR for so long

I have no idea about the number of betas, since I think are unclear on what constitutes a 1.0, I think we should get Vue 3 support working (it's WIP, hopefully public alpha soon) and call whatever we have at that point the 1.0 release.

Reply:
Uh are we dropping string templates entirely...?